ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -373,15 +370,15 @@ public void testUpdateContainerInAccount() throws JSONException {
     AccountBuilder accountBuilder = new AccountBuilder(account);
 
     // updating with different containers
-    for (int i = 0; i < 2; i++) {
+    for (int i = 0; i < CONTAINER_COUNT; i++) {
       Container container = account.getContainerById(refContainerIds.get(i));
       accountBuilder.removeContainer(container);
       ContainerBuilder containerBuilder = new ContainerBuilder(container);
       short updatedContainerId = (short) (container.getId() + 100);
```

Comment:
how do we know that +100 may not collide with any other existing container Ids ?